<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>getbus</class>
 <widget class="QDialog" name="getbus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>busbee</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-8</y>
     <width>610</width>
     <height>333</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>491</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 9pt &quot;Segoe UI&quot;;
color: grey;</string>
     </property>
     <property name="text">
      <string>                                       Please enter the valid information of your route </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Routefrom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="routeto">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>130</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>132</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RouteTo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>132</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RouteFrom:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Seatno">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>192</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SeatNo:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="date">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>190</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>192</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Date:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>250</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>250</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>viewTicket</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_username">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_routefrom">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_routeto">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_seatno">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_date">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="header">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>Ticket Information</string>
     </property>
    </widget>
    <widget class="QLabel" name="header_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>87</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>Username :</string>
     </property>
    </widget>
    <widget class="QLabel" name="header_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>RouteFrom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="header_4">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>Date :</string>
     </property>
    </widget>
    <widget class="QLabel" name="header_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>Routeto :</string>
     </property>
    </widget>
    <widget class="QLabel" name="header_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string>Seatno :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>290</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_logo">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>130</y>
       <width>211</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resourse.qrc">:/img/images/userloginlogo.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="header_7">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>240</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Tempus Sans ITC&quot;;</string>
     </property>
     <property name="text">
      <string> Thank You :)</string>
     </property>
    </widget>
    <zorder>header_4</zorder>
    <zorder>header</zorder>
    <zorder>header_2</zorder>
    <zorder>label_username</zorder>
    <zorder>header_3</zorder>
    <zorder>label_routefrom</zorder>
    <zorder>header_5</zorder>
    <zorder>label_routeto</zorder>
    <zorder>header_6</zorder>
    <zorder>label_seatno</zorder>
    <zorder>label_date</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>label_logo</zorder>
    <zorder>header_7</zorder>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources>
  <include location="Resourse.qrc"/>
 </resources>
 <connections/>
</ui>
